gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Thu Jul 21 11:44:22 CEST 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-all.zip
```

Comment:
I'd put in 7.4.2 directly

Reply:
Done. `7.4` was just the automatic suggestion by AS